src/v4l/mod.rs

```diff
@@ -0,0 +1,270 @@
+///
+/// # Neolink V4l
```

Comment:
This file appears to be about 95% copy-paste from the default stream command. The camera drive and setup loops are the same. Can we break those out into a common module and pass a "sink" trait to that module?  I think the Gstreamer module is already kinda structured that way, so it would be an easy change to the existing code.